pastel/client.go

```diff
@@ -102,6 +102,28 @@ func (client *client) Verify(ctx context.Context, data []byte, signature, pastel
 	return verify.Verification, nil
 }
 
+// ActTickets implements pastel.Client.ActTickets
+func (client *client) ActTickets(ctx context.Context, actType ActTicketType, minHeight int) (ActTickets, error) {
+	tickets := ActTickets{}
+
+	if err := client.callFor(ctx, &tickets, "tickets", "list", "act", actType, minHeight); err != nil {
+		return nil, errors.Errorf("failed to get id tickets: %w", err)
+	}
+
+	return tickets, nil
+}
+
+// RegTicket implements pastel.Client.RegTicket
+func (client *client) RegTicket(ctx context.Context, regTxid string) (*RegTicket, error) {
+	ticket := &RegTicket{}
+
+	if err := client.callFor(ctx, &ticket, "tickets", "get", regTxid); err != nil {
+		return nil, errors.Errorf("failed to get reg ticket: %w", err)
+	}
+
+	return ticket, nil
+}
+
 func (client *client) callFor(ctx context.Context, object interface{}, method string, params ...interface{}) error {
```

Comment:
This function makes me a bit confused, in my opinion, i think when call the callFor function, the seconds parameter should be a pointer, then for CallForWithContext, just need to pass the object itself, not a &object, i know maybe u want to check it the remote call has the reply, but i think it’s not a good idea to do like this, take the grpc method for example, we won’t check if the reply is nil. And i can see the cases for call the callFor are not consistent

Reply:
@alonlong  I didn't write this function so probably its not right for me to change what's in master. But happy to make the changes if @levkoburburas agrees with you. 
Personally, I think we cannot set parameter as something like *interface{} as its not allowed. And someone calling this function would have to "guess" that they need to pass pointer to an object which is risky.